pkg/localenvoy/controllers/ingress/envoyingress.go

```diff
@@ -90,7 +91,7 @@ func generateStatusHost(domain string, ingress *networkingv1.Ingress) string {
 		}
 	}
 
-	//TODO(jmprusi): Hardcoded to the first one...
+	// TODO(jmprusi): Hardcoded to the first one...
```

Comment:
nit: not really needed in this PR 😄 If you want to keep the diff scoped as small as possible, I'd revert this & the similar changes below... are they coming from code formatting?

Reply:
Ah yes, that my IDE that auto-formatted the file on save. I'll be careful next time.